tls/README.md

```diff
@@ -0,0 +1,382 @@
+# TLS registry
+
+This registry stores expected metadata for TLS artifacts in `openshift-*` namespaces. This metadata 
+is used as expected result in e2e test which validates cluster secrets and configmaps.
+
+## Registry purpose
+
+The registry is used to collect all TLS artifacts used in OpenShift, certificate key pairs and CA bundles alike. 
+For simplicity this document will use "certificate" for both certificate, key or CA bundle.
```

Comment:
```suggestion
For simplicity this document will use "pki artifact" for both certificate, key or CA bundle.
```

?  It's confusing to read certificate where it's actually a bundle.

Reply:
I really wanted to get away with this and just use "cert" throughout the text :/

Not sure if "pki artifact" is better, maybe "tls artifact"?